clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,24 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Replaces cuda_compile with it's equivalent SYCL repr
+macro(DPCT_COMPILE generated_files)
+  # list of dp.cpp source files
+  set(_sources "")
+
+  # segregate source files, library type and ignore nvcc options
```

Comment:
nvcc option should be filtered in migration,  here "and ignore nvcc options" is not needed.

Reply:
Removed.